src/checkers/flymake-collection-mypy.el

```diff
@@ -26,34 +26,98 @@
 
 ;;; Code:
 
+(require 'project)
 (require 'flymake)
 (require 'flymake-collection)
 
 (eval-when-compile
   (require 'flymake-collection-define))
 
+(defcustom flymake-collection-mypy-args
+  '("--show-column-numbers"
```

Comment:
Perhaps `--show-column-numbers` and other args that the regexps depend upon shouldn't be customizable?